cluster/src/main/scala/kanaloa/reactive/dispatcher/ClusterAwareBackend.scala

```diff
@@ -0,0 +1,90 @@
+package kanaloa.reactive.dispatcher
+
+import java.util.UUID
+
+import akka.actor._
+import akka.cluster.routing.{ClusterRouterGroup, ClusterRouterGroupSettings}
+import akka.routing._
+import akka.pattern.ask
+import akka.util.Timeout
+import kanaloa.reactive.dispatcher.ClusterAwareBackend.RouteeRef
+import scala.concurrent.Future
+import scala.concurrent.duration._
+
+class ClusterAwareBackend(
+  actorRefPath: String,
+  role:         String,
+  routingLogic: RoutingLogic = RoundRobinRoutingLogic()
+)(implicit
+  system: ActorSystem,
+  timeout: Timeout = 1.seconds) extends Backend {
+
+  private[dispatcher] lazy val router: ActorRef = {
+    val routerProps: Props = ClusterRouterGroup(
+      RoundRobinGroup(List("/user/" + actorRefPath)),
+      ClusterRouterGroupSettings(
+        totalInstances = 100,
```

Comment:
yeah, this is also a common practice for akka cluster. that total instance is only the upper limit and you just need to set it to be higher than the total number of instances of the node. So unless we have more than 100 instances of say "catalogService" this number should be high enough.  I am going to make it a parameter with defaultValue 100